docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1778,10 +1810,180 @@ And then in the YAML file for your Kubernetes deployment, specify the environmen
+            "ssl.protocol": "TLSv1.2",
+            "ssl.enabled.protocols": "TLSv1.2",
+            "ssl.endpoint.identification.algorithm": "HTTPS"
+        },
+        "kafkaServers": "********",
+        "kafkaApiKey": "********"
+    }
+```
+
+The service can map to the CADF format or the FHIR AuditEvent resource format by declaring a mapper type - 'cadf' or 'auditevent'. 
+
+- *CADF* Example
+```
+{
+    "action": "create",
+    "eventTime": "2020-12-10 16:49:22.307",
+    "eventType": "activity",
+    "id": "c62ee8f8-be77-49d0-aad0-c0bf52a05fdf",
+    "outcome": "success",
+    "typeURI": "http://schemas.dmtf.org/cloud/audit/1.0/event",
+    "tags": [
+    ],
+    "attachments": [
+        {
+            "contentType": "application/json",
+            "content": "rO0ABXQCUQp7CiAgICAicmVxdWVzdF91bmlxdWVfaWQiOiAiYzYyZWU4ZjgtYmU3Ny00OWQwLWFhZDAtYzBiZjUyYTA1ZmRmIiwKICAgICJhY3Rpb24iOiAiQyIsCiAgICAic3RhcnRfdGltZSI6ICIyMDIwLTEyLTEwIDE2OjQ5OjIyLjE2OCIsCiAgICAiZW5kX3RpbWUiOiAiMjAyMC0xMi0xMCAxNjo0OToyMi4zMDciLAogICAgImFwaV9wYXJhbWV0ZXJzIjogewogICAgICAgICJyZXF1ZXN0IjogImh0dHBzOi8vbG9jYWxob3N0Ojk0NDMvZmhpci1zZXJ2ZXIvYXBpL3Y0L1ZlcmlmaWNhdGlvblJlc3VsdCIsCiAgICAgICAgInJlcXVlc3Rfc3RhdHVzIjogMjAxCiAgICB9LAogICAgImRhdGEiOiB7CiAgICAgICAgInJlc291cmNlX3R5cGUiOiAiVmVyaWZpY2F0aW9uUmVzdWx0IiwKICAgICAgICAiaWQiOiAiMTc2NGQ4ZWEzMGEtYzUxOGMxOWItMWM0Zi00Yjc0LThhMDMtNGM5NjA0YmZkYjZlIiwKICAgICAgICAidmVyc2lvbl9pZCI6ICIxIgogICAgfSwKICAgICJldmVudF90eXBlIjogImZoaXItY3JlYXRlIiwKICAgICJkZXNjcmlwdGlvbiI6ICJGSElSIENyZWF0ZSByZXF1ZXN0IiwKICAgICJsb2NhdGlvbiI6ICIxMjcuMC4wLjEvbG9jYWxob3N0Igp9"
+        }
+    ],
+    "initiator": {
+        "id": "default@fhir-server",
+        "typeURI": "compute/machine",
+        "host": "192.168.86.20",
+        "credential": {
+            "token": "user-fhiruser"
+        },
+        "geolocation": {
+            "city": "Dallas",
+            "state": "TX",
+            "region": "US",
+            "annotations": [
+            ]
+        }
+    },
+    "observer": {
+        "id": "fhir-server",
+        "typeURI": "compute/node",
+        "name": "IBM FHIR Server - Audit",
+        "geolocation": {
+            "city": "Dallas",
+            "state": "TX",
+            "region": "US",
+            "annotations": [
+            ]
+        }
+    }
+}
+```
+
+- *AuditEvent* Example
+
+```
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://terminology.hl7.org/CodeSystem/audit-event-type",
+        "code": "rest",
+        "display": "Restful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/restful-interaction",
+            "code": "search",
+            "display": "search"
+        }
+    ],
+    "action": "E",
+    "period": {
+        "start": "2020-12-10T16:38:14.466Z",
+        "end": "2020-12-10T16:38:14.631Z"
+    },
+    "recorded": "2020-12-10T11:38:14.632173-05:00",
+    "outcome": "0",
+    "outcomeDesc": "success",
+    "purposeOfEvent": [
```

Comment:
is there supposed to be an actual purpose string in here somewhere.  having a "purpose" of "PurposeOfUse" seems pretty odd

Reply:
Interesting, apparently that *is* a valid value in the ValueSet.  Its a top-level code which has lots of potential children.  And its the *only* top-level code in the valueset.  So weird.

Reply:
Yeah - that's why I mentioned that it needs more work once we have a customer asking for a use-case